cloud/openstack/os_server.py

```diff
@@ -428,6 +440,7 @@ def _create_server(module, cloud):
     flavor = module.params['flavor']
     flavor_ram = module.params['flavor_ram']
     flavor_include = module.params['flavor_include']
+    reuse_fips = module.params['reuse_fips']
```

Comment:
I don't see where this is actually used. Later in this function you directly access `module.params['reuse_fips']`.

Am I missing something?